web/app.yaml

```diff
@@ -0,0 +1,27 @@
+runtime: nodejs10
+ 
+handlers:
+- url: /.*
+  static_files: dist/web/\1
+  upload: dist/web/.*
+
+- url: /(.*\.js)
+  redirect_http_response_code: 301
+  static_files: dist/web/\1
+  upload: dist/web/.*\.js
+
+- url: /(.*\.js.map)
+  redirect_http_response_code: 301
+  static_files: dist/web/\1
+  upload: dist/web/.*\.js.map
+
+- url: /favicon.ico
+  redirect_http_response_code: 301
+  static_files: dist/web/favicon.ico
+  upload: dist/web/favicon.ico
+
+
+- url: /.*
+  redirect_http_response_code: 301
+  static_files: dist/web/index.html
+  upload: dist/web/index.html
```

Comment:
Do we need these lines? I would expect the handler for `/.*` starting on line 4 to do everything we need.